core-rust/mesh-api-server/src/mesh_api/conversions/hashes.rs

```diff
@@ -1,5 +1,11 @@
 use crate::prelude::*;
 
-pub fn to_api_transaction_tree_hash(transaction_tree_hash: &TransactionTreeHash) -> String {
-    to_hex(transaction_tree_hash)
+pub fn to_api_hash_bech32m<T: IsTransactionHash>(
```

Comment:
Possibly `to_api_transaction_hash_bech32m` might be better?

Reply:
👍